unvendor_tests_from_wheel.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+
+# /// script
+# requires-python = ">=3.9"
+# dependencies = [
+#   "wheel",
+# ]
+# ///
+
+import fnmatch
+import os
+import shutil
+import subprocess
+import tempfile
+from itertools import chain
+from pathlib import Path
+
+
+def unvendor_tests(
+    install_prefix: Path,
+    test_install_prefix: Path,
+    retain_test_patterns: list[str] = [],
+) -> int:
+    """Unvendor test files and folders from installed package."""
```

Comment:
Since this is based on the `pyodide-build` implementation, should I conform to the license and add the notice here? The implementation is not unique, whatsoever,

Reply:
It's pretty small piece of functionality, so I would say no need for license info.